interactors/src/main/java/com/thefuntasty/interactors/disposables/DisposablesOwner.kt

```diff
@@ -1,497 +1,16 @@
-     * onComplete event. Empty lambda by default.
-     * @return disposable of internal [Maybe]. It might be used to
-     * dispose interactor when you need to dispose it in advance on your own.
-     */
-    fun <T : Any> Maybe<T>.executeStream(
-        onSuccess: (T) -> Unit,
-        onError: (Throwable) -> Unit = onErrorLambda,
-        onComplete: () -> Unit = { }
-    ): Disposable {
-        return subscribe(onSuccess, onError, onComplete).also {
-            disposables += it
-        }
-    }
-
-    /**
-     * Executes the [Completable] and adds its disposable to
-     * shared, automatically disposed, composite disposable.
-     *
-     * @param onComplete Lambda executed when internal [Completable] emits
-     * onComplete event
-     * @return disposable of internal [Completable]. It might be used to
-     * dispose interactor when you need to dispose it in advance on your own.
-     */
-    fun Completable.executeStream(onComplete: () -> Unit) = executeStream(onComplete, onError = onErrorLambda)
-
-    /**
-     * Executes the [Completable] and adds its disposable to
-     * shared, automatically disposed, composite disposable.
-     *
-     * @param onComplete Lambda executed when internal [Completable] emits
-     * onComplete event
-     * @param onError Lambda executed when internal [Completable] emits
-     * onError event
-     * @return disposable of internal [Completable]. It might be used to
-     * dispose interactor when you need to dispose it in advance on your own.
-     */
-    fun Completable.executeStream(
-        onComplete: () -> Unit,
-        onError: (Throwable) -> Unit = onErrorLambda
-    ): Disposable {
-        return subscribe(onComplete, onError).also {
-            disposables += it
-        }
-    }
-
-    @RestrictTo(RestrictTo.Scope.TESTS)
-    fun <T : Any> BaseFlowabler<T>.executeSubscriber(subscriber: TestSubscriber<T>) {
-        stream().subscribe(subscriber)
-
-        disposables += subscriber
-    }
-
-    @RestrictTo(RestrictTo.Scope.TESTS)
-    fun <T : Any> BaseSingler<T>.executeSubscriber(subscriber: TestSubscriber<T>) {
-        stream()
-            .toFlowable()
-            .subscribe(subscriber)
-
-        disposables += subscriber
-    }
-
-    @RestrictTo(RestrictTo.Scope.TESTS)
-    fun <T : Any> BaseMayber<T>.executeSubscriber(subscriber: TestSubscriber<T>) {
-        stream()
-            .toFlowable()
-            .subscribe(subscriber)
-
-        disposables += subscriber
-    }
-
-    @RestrictTo(RestrictTo.Scope.TESTS)
-    fun <T : Any> BaseCompletabler.executeSubscriber(subscriber: TestSubscriber<T>) {
-        stream()
-            .toFlowable<T>()
-            .subscribe(subscriber)
-
-        disposables += subscriber
-    }
-
-    @RestrictTo(RestrictTo.Scope.TESTS)
-    fun <T : Any> BaseObservabler<T>.executeSubscriber(observer: TestObserver<T>) {
-        stream().subscribe(observer)
-
-        disposables += observer
-    }
-}
+}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/abd629572327cbabcfd5483dec148fd3bc6b5ac2/interactors/src/main/java/com/thefuntasty/interactors/disposables/DisposablesOwner.kt#L16"></span>The file DisposablesOwner.kt is not ending with a new line.</td>
    </tr>
  </tbody>
</table>